verl/trainer/ppo/ray_trainer.py

```diff
@@ -954,6 +954,7 @@ def fit(self):
                             self.async_rollout_manager.sleep()
                         timing_raw.update(gen_batch_output.meta_info["timing"])
                         gen_batch_output.meta_info.pop("timing", None)
+                    continue
```

Comment:
Is this just for testing? Do you plan to remove this later? If so, please create an issue on your repository and add a comment with `TODO (Issue #): Remove continue`